tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java

```diff
@@ -31,26 +31,22 @@
   }
 
   // Back off types.
-  int copBuildTaskMaxBackoff = 5000;
-  int tsoMaxBackoff = 5000;
-  int scannerNextMaxBackoff = 40000;
-  int batchGetMaxBackoff = 40000;
-  int copNextMaxBackoff = 40000;
-  int getMaxBackoff = 40000;
-  int prewriteMaxBackoff = 20000;
-  int cleanupMaxBackoff = 20000;
-  int GcOneRegionMaxBackoff = 20000;
-  int GcResolveLockMaxBackoff = 100000;
-  int GcDeleteRangeMaxBackoff = 100000;
-  int rawkvMaxBackoff = 40000;
-  int splitRegionBackoff = 20000;
-  int commitMaxBackoff = 3000;
-
-  /** 20 seconds */
-  int batchPrewriteBackoff = 20000;
-
-  /** 3 seconds */
-  int batchCommitBackoff = 3000;
+  int seconds = 1000;
+  int copBuildTaskMaxBackoff = 5 * seconds;
```

Comment:
it's better to use `final static int COP_BUILD_TASK_MAX_BACK_OFF = ...`

Reply:
`final static` is useless in an interface type.

Reply:
u r right.